### PR TITLE
Ot208-57-58

### PR DESCRIPTION
**Pull request by acamus79**
It is interpreted that what you want to edit a slide is its main component, so a method to update the url of the slide image is implemented.

### CODE REVIEW

File: src/main/java/com/alkemy/ong/dto/request/slide/SlideUpdateDto.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com.alkemy.ong.dto.request.slide;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class SlideUpdateDto {
```
Comment (Oriana10) [inline src/main/java/com/alkemy/ong/dto/request/slide/SlideUpdateDto.java:10]: It could be called SlideResponseDto to keep a pattern in the nomenclature.

Comment (acamus79) [reply src/main/java/com/alkemy/ong/dto/request/slide/SlideUpdateDto.java:10]: It is a request DTO, and there is already a creation DTO called SlideRequestDto.

Comment (acamus79) [reply src/main/java/com/alkemy/ong/dto/request/slide/SlideUpdateDto.java:10]: Likewise, I consider the suggested change correct, so I refactor the name of SlideDto to SlideResponseDto. Thank you!

**Comment by MartinIriarte89** [comment]
Correct!

**Comment by EdwardDavys** [comment]
Done!

**Review by MartinIriarte89 — APPROVED** [review]

**Pull request merged**